devel/gdub/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        dougborg gdub 0.1.0 v
+github.setup        dougborg gdub ebe14f110153f05f3a50e7ff2eaa8d2704cde9d9
```

Comment:
You cannot just switch to a git commit like this, as it will change completely the form of the version string. If you really cannot convince upstream to make a release, then you will need to temporarily set the port version  by hand to something that makes sense, and when the port switches back to a release later on (say 0.1.1) will be correctly seen as a upgrade.

Reply:
One option, for instance, would be keep the version at 0.1.0, but bump the port revision to 1.

Another, would be to invent a new version, 0.1.0.1, such that when the next upstream release comes out, which will I presume be 0.1.1 or higher, it will be correctly seen as an update.

Reply:
The sad thing I don't want to clash with upstream version by accident.

Upstream release request is open since 2018-08-31: https://github.com/dougborg/gdub/issues/30, and it looks like it makes no sense to wait.

Reply:
Then I suggest we wait and see what the maintainer @breun suggests, as you need to use some sort of version that isnt just a git commit hash.

Reply:
FWIW I would go with 0.1.0.1, as it seems quite unlikely upstream would pick that for any future release, if that where ever to happen. Even it by some fluke it did, there is always the epoch to work around situations like this...

Reply:
I've added `version 0.1.0.1` below